build.cake

```diff
@@ -12,7 +12,7 @@ var configuration = Argument("configuration", "Release");
 //////////////////////////////////////////////////////////////////////
 
 var version = "3.10.0";
-var modifier = "";
+var modifier = ".2";
 
```

Comment:
I know........

Reply:
You seem to be using modifier in a different way than intended. Is this what you have done all along? Normally, I would have said version = "3.10.0.2" and modifier ="" (or "beta" or "rc-2" etc.).

Reply:
I use the modifier to get an increasing number, then I use the configuration Debug, which causes the script to add the suffix '-dbg'   I only use this during development, and clear this before commit, which I just forgot here.    If you set modifier = beta,  you get 3.10.0beta-dbg, note the missing dot between 0 and beta.

Reply:
I see, of course there's no problem with what you do in development.

With the modifiers, we actually used "-beta" or "-rc-2" when we were using them.